### PR TITLE
feature: backporting django-debug-toolbar from strawberry-django-plus

### PR DESCRIPTION
**Pull request by frleb**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [x] New feature
- [ ] Enhancement/optimization
- [x] Documentation

## Issues Fixed or Closed by This PR

* Fixes https://github.com/strawberry-graphql/strawberry-graphql-django/issues/138

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the CONTRIBUTING document.
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -29,6 +29,7 @@ classifiers = [
 python = ">=3.7,<4.0"
 Django = ">=3.2"
 strawberry-graphql = ">=0.154.0"
+django-debug-toolbar = { version = ">=3.4", optional = true }
```
Comment (bellini666) [inline pyproject.toml:32]: Probably also need to add a `tool.poetry.extras` section

**Review by bellini666 — CHANGES_REQUESTED** [review]
Awesome! Thank you for doing this

Just made a small code suggestion. Also, we need to make sure that the `strawberry_django` app can be added to `INSTALLED_APPS`, otherwise it won't be able to find the templates. Probably just need to define an `apps.py` file

File: pyproject.toml (hunk 1 above)
Comment (frleb) [reply pyproject.toml:32]: done !

**Review by bellini666 — APPROVED** [review]
LGTM 👍🏼 

Just a question, did you test that this is indeed working? For example by running the example project?

**Comment by frleb** [comment]
> LGTM 👍🏼
> 
> Just a question, did you test that this is indeed working? For example by running the example project?

Good point, I retested by cloning my branch again. I had to force the addition of the debug_toolbar in the poetry of the demo project

**Comment by blueyed** [comment]
It should also copy over/adjust the documentation then: https://blb-ventures.github.io/strawberry-django-plus/debug-toolbar/ (https://github.com/blb-ventures/strawberry-django-plus/blob/90fa11f4c5254a1fba0db60a9a4362568b11cbcc/docs/debug-toolbar.md).

**Review by bellini666 — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**